llvm/lib/Transforms/Utils/CloneFunction.cpp

```diff
@@ -238,6 +238,13 @@ void llvm::CloneFunctionInto(Function *NewFunc, const Function *OldFunc,
       }
     }
 
+    // Avoid cloning local variables of subprograms that won't be cloned.
```

Comment:
Can you add a sentence explaining why some subprograms won't be cloned? Is it because they are inlined and a copy already exists?